src/app/containers/Brand/index.test.jsx

```diff
@@ -42,5 +77,42 @@ describe(`BrandContainer`, () => {
 
       expect(skipLinkHref).toBe('#content');
     });
+
+    describe('preffered variant cookie', () => {
+      let container;
+
+      beforeEach(() => {
+        container = render(
+          BrandContainerWithContext({
+            ...variantServiceContextStub,
+          }),
+        ).container;
+      });
+
+      afterEach(() => {
+        document.cookie = '';
+        jest.clearAllMocks();
+      });
+
+      it('should be set when ScriptLink is clicked and cookie is not defined', () => {
+        document.cookie = '';
+
+        const scriptLink = container.querySelector('a[data-variant="test"]');
+        fireEvent.click(scriptLink);
+
+        expect(spy).toHaveBeenCalledTimes(1);
+        expect(document.cookie).toEqual('; ckps_news=test');
+      });
+
+      it('should be updated when ScriptLink is clicked and cookie exists', () => {
+        document.cookie = 'ckps_news=lat';
+
+        const scriptLink = container.querySelector('a[data-variant="test"]');
+        fireEvent.click(scriptLink);
+
+        expect(spy).toHaveBeenCalledTimes(1);
+        expect(document.cookie).toEqual('; ckps_news=test');
+      });
```

Comment:
The two tests are common since they are checking for the update of `cookie`. Could we combine this to one test instead of two.

Reply:
I think it makes sense to leave the test as it is right now because it is more readable. Currently as it is, its very clear that clicking on the ScriptLink should set cookie when one is not defined and when one is defined - which is slightly harder to see if it's all one test